modules/@apostrophecms/ui/ui/apos/components/AposButton.vue

```diff
@@ -33,7 +33,7 @@
         />
         <slot name="label">
           <span class="apos-button__label" :class="{ 'apos-sr-only' : (iconOnly || type === 'color') }">
-            {{ $t(label) }}
+            {{ $t(label, interpolate) }}
```

Comment:
I also introduced interpolate support for AposButton as that is sometimes needed.